.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -25,7 +25,7 @@ body:
   - type: textarea
     attributes:
       label: To Reproduce
-      description: Steps to reproduce the behavior. Please paste the code and the output as text, and avoid using screenshots. Bonus points if the steps only include SQL queries.
+      description: Steps to reproduce the behavior. Please paste the code and the output as text, and avoid using screenshots. Please add the required imports for Python scripts (e.g., import duckdb). Bonus points if the steps only include SQL queries.
```

Comment:
Import versions would also be good

Reply:
I can deal with that but adding `import pandas as pd`, numpy, etc. can be tedious.